internal/model/model.go

```diff
@@ -84,8 +84,12 @@ type service interface {
 }
 
 type Model struct {
-	cfg             *config.Wrapper
-	db              *leveldb.DB
+	cfg *config.Wrapper
+	db  *leveldb.DB
+
+	cache    *db.NamespacedKV
+	cacheMut sync.Mutex
```

Comment:
It's more of a OCD thing to me I guess. It's not a bottleneck now but it might be due to the poor code I introduce.